terraform/modules/api_gateway/gateway/main.tf

```diff
@@ -73,7 +73,7 @@ module "query_post" {
   authorizer_id = aws_api_gateway_authorizer.api_key.id
   api_resource  = module.query_resource.aws_api_gateway_resource
 
-  require_api_key = false
+  require_api_key = true
```

Comment:
I don't see a terraform module here for zonal statistics, similar to these 'query_post' and 'query_get' modules.  Do we need a terraform module for zonal statistics as well to actually enable rate limiting for zona statistics?

Reply:
Good catch, that's a deprecated endpoint and not used much looking at the usage stats. How about I merge this and address that in future iteration.